File: GC.Domain/Gardens/GardenerBlank.cs
using GC.Domain.Persons;
using System;
using System.ComponentModel.DataAnnotations;

namespace GC.Domain.Gardens
{
    public class GardenerBlank: FromPersonBlank
    {
        public Guid? Id { get; set; }

        [Required(ErrorMessage = "Укажите имя садовода")]
        public String? FirstName { get; set; }
        public String? MiddleName { get; set; }

        [Required(ErrorMessage = "Укажите фамилию садовода")]
        public String? LastName { get; set; }

        [Required(ErrorMessage = "Укажите участок садовода")]
        public Guid? SectorId { get; set; }
    }
}

File: GC.WebSpace/ClientApp/domain/gardens/gardeners/gardenerBlank.ts
import { Gardener } from "./gardener";

export class GardenerBlank {
    constructor(
        public id: string | null,
        public firstName: string,
        public middleName: string | null,
        public lastName: string,
        public sectorId: string | null
    ) { }

    static create = (gardener: Gardener): GardenerBlank => {
        return new GardenerBlank(gardener.id, gardener.firstName, gardener.middleName, gardener.lastName, gardener.sectorId);
    }

    static empty = (): GardenerBlank => {
        return new GardenerBlank(null, '', '', '', null);
    }
}